docs/src/main/paradox/cassandra.md

```diff
@@ -120,7 +120,7 @@ To use @extref[Akka Discovery](akka:discovery/) make sure the `akka-discovery` d
   version=PekkoVersion
 }
 
-To enable @extref[Akka Discovery](akka:discovery/) with the @apidoc[DefaultSessionProvider], set up the desired service name in the discovery mechanism of your choice and pass that name in `service-discovery.name`. The example below extends the `alpakka.cassandra` config section and only overwrites the service name.
+To enable @extref[Apache Pekko Discovery](pekko:discovery/) with the @apidoc[DefaultSessionProvider], set up the desired service name in the discovery mechanism of your choice and pass that name in `service-discovery.name`. The example below extends the `alpakka.cassandra` config section and only overwrites the service name.
```

Comment:
reference to `alpakka.cassandra` here intended?

Reply:
I'm not sure if we've changed this repo to replace the `alpakka` in the config names. I think there are plenty of these alpakka configs in the docs still (too).

Reply:
I raised https://github.com/apache/incubator-pekko-connectors/issues/25 -- we are still using alpakka in the config names so best to wait for that issue to update the config names in the docs